packages/tsconfig-reference/copy/en/options/jsx.md

```diff
@@ -6,11 +6,11 @@ oneline: "Specify what JSX code is generated."
 Controls how JSX constructs are emitted in JavaScript files.
 This only affects output of JS files that started in `.tsx` files.
 
-- `react`: Emit `.js` files with JSX changed to the equivalent `React.createElement` calls
-- `react-jsx`: Emit `.js` files with the JSX changed to `_jsx` calls
-- `react-jsxdev`: Emit `.js` files with the JSX changed to `_jsx` calls
+- `react-jsx`: Emit `.js` files with the JSX changed to `_jsx` calls for production
+- `react-jsxdev`: Emit `.js` files with the JSX changed to `_jsx` calls for development
```

Comment:
```suggestion
- `react-jsx`: Emit `.js` files with the JSX changed to `_jsx` calls optimized for production
- `react-jsxdev`: Emit `.js` files with the JSX changed to `_jsx` calls for development only
```

that makes more sense since `react-jsx` can be used in dev as well. It's just that `react-jsxdev` will crash in prod.